### DISCUSSION TITLE
Support set session settings

### DISCUSSION THREAD

**Discussion by guoxiaod** [grafana/clickhouse-datasource#354] [General]
We want to use some clickhouse experimental features, such as window function, but we can not set session settings in current version.

1. sql 
`set allow_experimental_window_functions = 1;`
2. or add settings in clickhouse.OpenDB ( clickhouse.Options ):
```
+               Settings: clickhouse.Settings{
+                       "allow_experimental_window_functions": 1,
+               },
```

**Comment by mshustov** [comment]
ClickHouse settings can be configured on the connection level starting from `v3.1.0` / `v2.2.1`
![2023-06-04_14-36-20](https://github.com/grafana/clickhouse-datasource/assets/3198181/6761969f-37ed-4be9-8c39-c303d0a05d5b)
